### PR TITLE
Add unknown change resp stmt

### LINKED ISSUE #52: schematron warnings
At the current state the sample encodings for MEI v4.0.0 throw 266 schematron warning on validation theses are:
- mainly: It is recommended that the agent responsible for the change be recorded in a resp attribute or in a name, corpName, or persName element in the respStmt element.
- some: The value in @resp should correspond to the @xml:id attribute of an element within the metadata header.
- some: At least one element pair (a resp element and a name-like element) is recommended. Alternatively, each name-like element may have a @role attribute.

Should we try fix those or ignore them, as they are merely warnings?

### LINKED ISSUE #84: Fix missing responsibilities in change elements

### PR DESCRIPTION
**Pull request by bwbohl**
closes #84 
closes #52

### CODE REVIEW

**Comment by musicEnfanthen** [comment]
You can simply change the target branch in gh when editing the pr title 😉

**Comment by bwbohl** [comment]
oh wow, thx for the great hint !!

**Review by rettinghaus — APPROVED** [review]

File: MEI_4.0/Music/Complete_examples/Weber_Arie.mei [hunk 1]
```diff
@@ -166,6 +180,9 @@
     </workList>
     <revisionDesc>
       <change n="1">
+        <respStmt>
+          <name></name>
```
Comment (bwbohl) [inline MEI_4.0/Music/Complete_examples/Weber_Arie.mei]:
```suggestion
          <name>Unknown</name>
```

as surfaced in review of #83 by @musicEnfanthen

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Thank you!

**Pull request merged**